vm/safepoints.cpp

```diff
@@ -35,11 +35,18 @@ void factor_vm::handle_safepoint(cell pc) {
   faulting_p = false;
 
   if (atomic::load(&safepoint_fep_p)) {
+    atomic::store(&safepoint_fep_p, false);
```

Comment:
@AlexIljin is there a reason you can't just ``enqueue_fep()`` and then ``t`` to throw an exception into Factor?  Then we wouldn't need the ``stop_on_ctrl_break`` boolean.